test/components/geojson_test.dart

```diff
@@ -32,6 +36,24 @@ main() {
       expect(() => Position.of([1, 2, 3, 4]).toList(),
           throwsA(isA<AssertionError>()));
     });
+
+    test('Position vector addition', () {
+      expect(Position(1, 2, 3) + Position(1, 2, 3), Position(2, 4, 6));
+    });
```

Comment:
should you also test that you can properly add two high value real world lat/lng and have it properly wrap around? (same for subtraction, and perhaps cross product and dot product?)

Reply:
we definitely should

Reply:
depends on a decision about the coordinate normalization logic